specification/search/resource-manager/Microsoft.Search/stable/2015-02-28/search.json

```diff
@@ -118,6 +118,55 @@
             }
           }
         }
+      },
+      "post": {
```

Comment:
We're adding this to 2015-02-28 too?

Reply:
I noticed that it was implemented for all api versions, so I thought the documentation should match.

Reply:
It probably shouldn't be implemented for all API versions.

In general, we don't retroactively update Swagger for old API versions. No harm in doing it, it's just extra maintenance that doesn't usually need to be done.